sdk/search/search/README.md

```diff
@@ -400,6 +400,121 @@ for (const result of updateResult.results) {
 }
 ```
 
+### 3. Create and authenticate a `SearchServiceClient`
+
+You could use the following code to create and authenticate a `SearchServiceClient`. 
+
+```js
+import { SearchServiceClient, SearchApiKeyCredential }  from "@azure/search";
+
+const endPoint: string = "<endpoint>";
+const apiKey: string = "<apikey>";
+
+const credential: SearchApiKeyCredential = new SearchApiKeyCredential(apiKey);
+const client: SearchServiceClient = new SearchServiceClient(
+  endPoint,
+  credential
+);
+```
+
+#### Examples
+
+#### Get a list of existing indexes in the service
+
+```js
+let indexes:Index[] = [];
+indexes = await client.listIndexes();
+indexes.forEach(idx => {
+  console.log(idx.name);
+  idx.fields.forEach(idxField => {
+    console.log(idxField.name);
+  });
+});
+```
+
+#### Get a list of existing skillsets in the service
+```js
+let skillsets:Skillset[] = [];
+skillsets = await client.listSkillsets();
+skillsets.forEach(set => {
+  console.log(`Name: ${set.name}`);
+  console.log(`Desc: ${set.description}`);
+  console.log(`Skills`);
+  set.skills.forEach(skill => {
+    console.log(`  odatatype: ${skill.odatatype}`)
+    console.log(`  Name: ${skill.name}`)
+    console.log(`  Description: ${skill.description}`)
+  });
+});
+```
+
+#### Get a list of existing synonymMaps in the service
+```js
+let sms:SynonymMap[] = [];
+sms = await client.listSynonymMaps();
+sms.forEach(elem => {
+  console.log(elem.name);
+});
+```
+
+#### Create an Index
+```js
+const idxCreated:Index = await client.createIndex({
+  analyzers: [],
+  charFilters: [],
+  defaultScoringProfile: "",
+  scoringProfiles: [],
+  suggesters: [],
+  tokenFilters: [],
+  tokenizers: [],
+  name: `azureblob-index998`,
+  fields: [{
+    name: "content",
```

Comment:
perhaps call this `id` or something since it's the key? I'd also like to see other field types, like a complex type or a collection type, and some flags for searchable, facetable, etc that are different from the defaults. I can add this example in my doc PR if you prefer

Reply:
Changed it. Also added optional params such as suggesters and analyzers. Please let me know if this is sufficient.